edenai_apis/llmengine/llm_engine.py

```
@@ -868,6 +851,22 @@ async def amoderation(
             standardized_response=standardized_response,
         )
 
+    def llm_moderation(
+        self, input: Union[str, List], **kwargs
```

Ruff (0.15.9)

[error] Function argument `input` is shadowing a Python builtin

(A002)